.circleci/ci-packages.txt

```diff
@@ -256,3 +256,4 @@ python3-setuptools-40.5.0
 python3-yamllint-1.22.1
 ShellCheck-0.7.1
 sqlite3-3.28.0
+docker
```

Comment:
Note that this file is generated/modified by `.circleci/build_dependencies.sh`. I'm not sure whether *only* adding a dependency here sustains in the long run.

Reply:
Do you think that adding the installation of the docker directly to the build_dependencies.sh should be the right option?

Reply:
Likely - but I can not say for sure. I had to experiment with it myself as well.

Reply:
This is based on a suggestion I made. It should be fine this way. IIUC the dependency check will or can add a specific version number.

Reply:
This is not about the absence of the version number but about the absence of the package (or a package pulling the package) elsewhere. Since this file is generated by `.circleci/build_dependencies.sh` I assume on the next run of that script it might vanish again.

Reply:
Correct. I have mentioned `dependencies.yaml` in https://github.com/os-autoinst/openQA/pull/3623#pullrequestreview-546353253 as well